api/models/Group.js

```diff
@@ -750,15 +752,16 @@ module.exports = bookshelf.Model.extend(merge({
     .then(g => {
       var creator = g.relations.creator
       var recipient = process.env.NEW_GROUP_EMAIL
+      const locale = creator.get('settings')?.locale || 'en'
       return Email.sendRawEmail(recipient, {
-        subject: "New Hylo Group Created: " + g.get('name'),
-        body: `Group
-          Name: ${g.get('name')}
+        subject: locales[locale].groupCreatedNotifySubject(g.get('name')),
+        body: `${locales[locale].Group()}
+          ${locales[locale].Name()}: ${g.get('name')}
           URL: ${Frontend.Route.group(g)}
-          Creator Email: ${creator.get('email')}
```

Comment:
this is an email that goes to us as Hylo stewards telling us about new groups, it really doesnt need to be translated. but this is ok